lib/ioki/model/operator/public_transport_stop.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class PublicTransportStop < Base
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :city,
+                  type: :string,
+                  on:   :read
+
+        attribute :country,
+                  type: :string,
+                  on:   :read
+
+        attribute :county,
+                  type: :string,
+                  on:   :read
+
+        attribute :current_arrival,
+                  type: :string,
```

Comment:
```suggestion
                  type: :date_time,
```

Haven't tested it, but maybe we want to have a `DateTime` object here?

In the passenger API we defined it as `string` as well. Maybe it's more comfortable for you to have a `DateTime` object? Don't really have an opinion here, whatever is easier for you.

Reply:
Oh that's my bad. Of course these should be of type `date_time`. Thanks 🙇